nakadi/swagger.yaml

```diff
@@ -1,45 +1,33 @@
 swagger: '2.0'
 info:
-  title: Nakadi
+  title: Nakadi API Definition
   description: |
     -------------------------
-    API for nakadi operations
+    Nakadi API Definition
     -------------------------
 
-    A few concepts in this working draft:
-
+    * Low-level API
     * Stantardised event format (see #/definitions/Event);
+    * Partition strategy: event store will infer the partition from the `partitioning_key` in the message body;
 
-    * Partition resolution: event store will infer the partition where a message
-    needs to be set looking into the `partitioning_key` in the message body;
-
-    * High level API should support automatic redistribution of the event streams
-    during client topology changes.
-
-  version: '0.3'
+  version: '0.4'
   contact:
-    name: Team Aruha
-    email: team-aruha@zalando.de
+    name: Team Aruha @ Zalando
+    email: nakadi-users@zalando.de
```

Comment:
Why you changed it? I actually thought that nakadi-users@zalando.de is mailing list our stakeholders will be subscribed to.